Python/Algorithms/aditya/binary_tree.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABCMeta, abstractmethod
+from collections import deque
```

Comment:
The idea is to use the queue that we have created in our master branch or in your branch

Reply:
Ok , have changed that to use my own stack and Queue implementation .